src/EditorFeatures/Core/EditorFeaturesResources.resx

```diff
@@ -878,4 +878,8 @@ Do you want to proceed?</value>
   <data name="Operator_Overloaded" xml:space="preserve">
     <value>Operator - Overloaded</value>
   </data>
+  <data name="Operation_is_not_ready_for_0_yet_see_task_center_for_more_detail" xml:space="preserve">
```

Comment:
1. 'See' should be capitalized to match loc string.
2. do you have examples of waht would go in the `{0}` section?

Reply:
it is name of document or project. it is in the code. are you going to ask every single line which is already in code?

Reply:
> are you going to ask every single line which is already in code?

I don't know what that means.

Reply:
see where the resrouce is used. you can find it by simple search in browser.

Reply:
you have all the code right there in PR. not sure why you are asking, best way to look the code.

Reply:
>ou have all the code right there in PR. not sure why you are asking,

Because i wanted to get an answer.  It wasn't clear to me.

Reply:
I don't understand why you need to start a meta conversation about lines of questioning in a PR.  I ask on a PR because i want to understand something better.

Reply:
since it felt like you are asking like why are you using "a" here when answer is right there. it is not like you are new to Roslyn or C#. simple text search give you the answer right away and it doesn't involve anything complex.

Reply:
>  it is not like you are new to Roslyn or C#.

Right.  Which is why i asked.  I ask questions for things that i'm looking for *clarity* on.  It means that it isn't clear to me from my reading of the PR.  

> since it felt like you are asking like why are you using "a" here when answer is right there. 

It wasn't clear to me.  So i asked.  That's how i always do things in PRs.  Thank you for clarifying things. Now can we please move on?    I literally do not understand why every PR with you has to have some sort of metadiscussion where you either complain aobut my questions, complain that my points are too brief, or compain that i'm 'throwing words around'.  

I'm simply reviewing the PR and bringing up things are are concerns/questions i have.  My goal is to get the best possible change made to the codebase, and i like to work with and engage the team and PR author to get to that point.  I don't know why this has to eb explained in each and every PR with you and you alone.

Reply:
the review we are doing right now is fine. I am fine doing review we are doing right now, but the intial comment you left, I am not sure why you are leaving such comments. that's not PR code review. that kind of design question, like I said, open new issue and discuss them there. if people agree, they will create new PR to address those design changes. otherwise, people will keep existing design.